fdbclient/admin_client.go

```diff
@@ -817,13 +823,17 @@ func (client *cliAdminClient) GetBackupStatus() (*fdbv1beta2.FoundationDBLiveBac
 }
 
 // StartRestore starts a new restore.
-func (client *cliAdminClient) StartRestore(url string, keyRanges []fdbv1beta2.FoundationDBKeyRange) error {
+func (client *cliAdminClient) StartRestore(url string, keyRanges []fdbv1beta2.FoundationDBKeyRange, encryptionKeyPath string) error {
 	args := []string{
 		"start",
 		"-r",
 		url,
 	}
 
+	if encryptionKeyPath != "" {
```

Comment:
I believe the encryption feature was added in 7.3 (but I might be wrong) could you check this and if that's the case add a validation step here to ignore this flag for clusters running on 7.1?